### ISSUE TITLE
Next suggestion isn't read

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#17400]
Discovered by -jooyoungseo

1. Trigger an inline suggestion
2. Trigger the hover
3. Trigger the next suggestion using the keybinding
4. 🐛 the screen reader doesn't read the next suggestion. The hover has to be closed and reopened for it to be read

cc -isidorn
